src/psyclone/tests/psyGen_test.py

```diff
@@ -1826,8 +1826,13 @@ def test_node_dag(tmpdir):
                  "<title>loop_0_end</title>",
                  "<title>kernel_testkern_qr_code_2</title>",
                  "<title>kernel_testkern_qr_code_4</title>",
-                 "<svg", "</svg>", "blue", "green", "red"]:
+                 "<svg", "</svg>", ]:
         assert name in result
+    for colour_name, colour_code in [("blue", "#0000ff"),
+                                     ("green", "#00ff00"),
+                                     ("red", "#ff0000")]:
+        assert colour_name in result or colour_code in result
```

Comment:
Nice generalisation to solve problem of graphviz producing hex codes instead of names.